README.md

```diff
@@ -160,7 +637,9 @@ create the following staging resources.
 
 See [API Docs](./api-docs/README.md) for more details.
 
-## Building locally
+## Contributing
```

Comment:
CONTRIBUTING.md probably, this README is getting big.

Reply:
I'm going to leave this as a todo. I'll tackle creating a true contributing guide as a separate thing.